src/views/settings/Applet/AppletHost/AppletHostDetail/Applets.vue

```diff
@@ -77,7 +84,7 @@ export default {
             label: this.$t('Date')
           },
           actions: {
-            width: '160px',
+            width: '138px',
             formatterArgs: {
               hasUpdate: false,
               hasDelete: false,
```

Comment:
I recommend taking a closer look at the exported functions, classes, and components to understand their purpose, structure and usage. For instance:

1. Check if all methods are used correctly and efficiently.

2. Analyze how data is passed between related resources.

3. Review the use of external libraries that may require updates.

4. Optimize code by eliminating redundancies, improving readability with comments or other tools.

If you find any irregularities or potential issues while examining these elements, I would suggest revising them according to your requirements. However, the provided text does not contain errors based on current knowledge or context available up until October 2021. Please let me know more about what specific points you want detailed feedback on!